pkgs/by-name/pl/plakar/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  fuse,
+}:
+buildGoModule rec {
+  pname = "plakar";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    tag = "v${version}";
+    hash = "sha256-peyATyUJe6EFvQutlBLpLjyXputdoKPQmqW8pYeXiVI=";
+  };
+
+  vendorHash = "sha256-r8zE1mwoPOcsMeUsm1Rfq5XQ6Rze/gmcMwilzfM4xOk=";
+
+  buildInputs = [
+    fuse
+  ];
+
+  # mount: fusermount: exec: "fusermount": executable file not found in $PATH
+  checkPhase = ''
+    go test -skip TestExecuteCmdMountDefault ./...
+  '';
```

I've seen other go based packages using the following pattern, while more verbose when skipping a single test, I think it provides more options for the future should specific tests fail on specific platforms.
```suggestion
  checkFlags =
    let
      skippedTests = [
        # mount: fusermount: exec: "fusermount": executable file not found in $PATH
        "testExecuteCmdMountDefault"
      ];
    in
    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
```